website/content/partials/datasource/hcp-packer-image/Config-not-required.mdx

```diff
@@ -0,0 +1,19 @@
+<!-- Code generated from the comments of the Config struct in datasource/hcp-packer-image/data.go; DO NOT EDIT MANUALLY -->
```

Comment:
This is a new file that has to be included in the data source main `mdx` file to appear in the docs.